audit/projects/k8s-staging-addon-manager/iam.json

```diff
@@ -19,12 +19,6 @@
       ],
       "role": "roles/cloudbuild.serviceAgent"
     },
-    {
-      "members": [
-        "serviceAccount:service-103321906213@container-analysis.iam.gserviceaccount.com"
-      ],
-      "role": "roles/containeranalysis.ServiceAgent"
-    },
```

Comment:
Expected, this is https://github.com/kubernetes/k8s.io/pull/2697

Reply:
Also, confirmation that https://github.com/kubernetes/k8s.io/issues/1963 is complete